README.md

```diff
@@ -26,6 +26,23 @@ The `azdev` tool is designed to aid new and experienced developers in contributi
     - For Azure CLI: https://github.com/Azure/azure-cli
     - For Azure CLI Extensions: https://github.com/Azure/azure-cli-extensions
     - Any other repository that you might have access to that contains CLI extensions.
+
+    After forking `azure-cli`, follow the below commands to set up:
+    ```Shell
+    # Clone your forked repository
+    git clone git@github.com:<your-github-name>/azure-cli.git
+    cd azure-cli
+    # Add the Azure/azure-cli repository
+    git remote add upstream git@github.com:Azure/azure-cli.git
+    # Reset the default dev branch to track dev branch of Azure/azure-cli so you can use it to track the latest azure-cli code.
+    git branch dev -u upstream/dev
+    # Develop with a new branch
+    git checkout -b feature_branch
+    ...
```

Comment:
is it possible to redirect to authoring command part here?

Reply:
and it would will be better to make sure virtual environment activated.
Could we move step 3 in front of this part, at least before coding and pushing back.

Reply:
It makes sense to create the virtual env first before command authoring. But this part was originally for cloning code and branch management. I did not intend to mention command authoring in this part, so used `...`. I only want to emphasis the `feature_branch` should be push to the `origin` remote as some developers push to `upstream` and found they have no permission to do so. Maybe I can move `git push -u origin feature_branch` to the `Authoring commands and tests` part.